### ISSUE TITLE
Routes generator produces ill-indented code

### ISSUE THREAD

**Issue by fwolfst** [thoughtbot/administrate#505]
Currently (0.1.4) administrate creates routes like the following (if more than one resource is found):

```
  namespace :admin do
    resources :departments
resources :services
resources :service_types
resources :usages

    root to: "departments#index"
  end
```

I expect it to create well indented code like

```
  namespace :admin do
    resources :departments
    resources :services
    resources :service_types
    resources :usages

    root to: "departments#index"
  end
```

**Comment by carlosramireziii** [comment]
This still seems to be an issue in `0.4.0`. Anyone want to take a stab at it?

**Comment by fwolfst** [comment]
I am not sure anymore, but my changes might just have fixed the issue (dont know why I didnt file a PR though).

**Comment by nickcharlton** [comment]
I think this was fixed in #506, so I'm going to close this. If it's not, please reopen!
